ibis/backends/tests/conftest.py

```diff
@@ -192,6 +192,13 @@ def df(alltypes):
     return alltypes.execute()
 
 
+@pytest.fixture(scope='session')
+def pandas_df(backend, alltypes):
```

Comment:
maybe call this `pandsa_alltypes`?

Reply:
I called it `pandas_df` since the regular fixture called  `df` across all the test code.